docs/content/documentation/modules/tensor_mechanics/C0TimoshenkoBeam.md

```diff
@@ -0,0 +1,192 @@
+#C0 Timoshenko Beam Element
+
+A beam element (\ref{fig:beam}) is used to model the response of a structural element that is long in one dimension compared its cross-section. There are two popular formulation of beam elements:
+
+ !media media/tensor_mechanics_beam_representation.png width=40% margin-left=30px float=right id=fig:beam caption= Beam element with 2 nodes and 3 translational and 3 rotational degrees of freedom at each node.
+
+* Euler-Bernoulli beam element: used to model bending deformation in long and slender beams. The two main assumptions in this beam theory are that: (i) the beam cross-section is rigid and does not deform under the application of transverse or lateral loads, and (ii) the cross-section of the beam remains planar and normal to the deformed axis of the beam.
+
+* Timoshenko beam element \citep{timoshenko_correction_1921, timoshenko_transverse_1922}: used to model both shear and bending deformation in short and thick beams. The beam cross-section does not deform in this beam theory as well and it remains planar. But the cross-section need not be normal to the deformed axis of the beam. Euler-Bernoulli beam element can be derived as a special case of the Timoshenko beam element.
+
+Therefore, a C0 timoshenko beam element is implemented in MOOSE. This element has two nodes and each node has 6 degrees of freedom (dof) - 3 translational and 3 rotational displacements. All the 12 dofs are considered to be independent and the variation of both translational and rotational displacements along the length of the beam are modeled using first order Lagrange shape functions. The independent rotational dofs at the nodes makes it easier to model the shear deformation which results in non-perpendicular cross-sections with respect to the beam axis.
```

Comment:
Capitalize 'Timoshenko'.  Also, I think it's better to capitalize 'dof', but that's up to you.